int64 mode_id
---
bool success
string message